OpenRA.Mods.Common/Traits/Upgrades/DeployToUpgrade.cs

```diff
@@ -28,6 +28,9 @@ public class DeployToUpgradeInfo : ITraitInfo, Requires<UpgradeManagerInfo>
 			"Leave empty to allow any.")]
 		public readonly HashSet<string> AllowedTerrainTypes = new HashSet<string>();
 
+		[Desc("Can this actor deploy on slopes?")]
+		public readonly bool CanDeployOnRamps = false;
```

Comment:
It is just not redundant which is actually better. Explaining terms with the same word isn't helping.